content/en/docs/zero-code/java/spring-boot/_index.md

```diff
@@ -419,21 +419,54 @@ springBoot {
 
 Automatic instrumentation is available for several frameworks:
 
-| Feature        | Property                                        | Default Value |
-| -------------- | ----------------------------------------------- | ------------- |
-| Logback        | `otel.instrumentation.logback-appender.enabled` | true          |
-| Spring Web     | `otel.instrumentation.spring-web.enabled`       | true          |
-| Spring Web MVC | `otel.instrumentation.spring-webmvc.enabled`    | true          |
-| Spring WebFlux | `otel.instrumentation.spring-webflux.enabled`   | true          |
+| Feature               | Property                                        | Default Value |
+|-----------------------|-------------------------------------------------|---------------|
+| `@WithSpan`           | `otel.instrumentation.annotations.enabled`      | true          |  todo should this be here or in section below?
```

Comment:
@jeanbisutti I think it might be good to have an overview of all instrumentations here